lib/libspl/include/sys/simd.h

```diff
@@ -551,7 +551,7 @@ zfs_sha512_available(void)
 
 #elif defined(__powerpc__)
 
-#define	kfpu_allowed()		1
+#define	kfpu_allowed()		0
```

Comment:
I think this is the most importing part ... as long as we have no `kfpu_begin()` and `end()` currently.

Reply:
We tried just this change, and it didn't help.  I think this just guards the power8 code, not the general sha*-ppc code.

Reply:
Ah yes, I will fix this in some pull request later, when the upcoming FreeBSD runners can check this change.

Reply:
How would you guys like to proceed with this change?

Reply:
We have two options:
1) setting `kfpu_allowed()` to zero - this is fine and works
2) guard also sha*-ppc (but I currently do now know what is the minimum fpu spec to be used)

So with this patch the generic powerpc problem will go away, and I have time to figure out the correct "`kfpu_allowed()` + whatever check" ...

Reply:
If you want to split this into two changes (kfpu_allowed() and the ELFv2 BE), I'm fine with that.  I pushed this change to FreeBSD as a stopgap until it's fixed here and we can pull the update.  The ELFv2 BE change works correctly on FreeBSD.

Oh, reviewing this again, the kfpu_allowed change actually needs to be made in include/os/freebsd/spl/sys/simd_powerpc.h, too.  I'll update the patch to include that.

Reply:
Sounds good.  Then I'll go ahead and pull in the updated patch.  I don't think we need to worry about splitting it.